source/common/http/utility.cc

```diff
@@ -80,6 +80,9 @@ std::string Utility::parseCookieValue(const HeaderMap& headers, const std::strin
       for (const std::string& s : StringUtil::split(std::string{header.value().c_str()}, ';')) {
         size_t first_non_space = s.find_first_not_of(" ");
         size_t equals_index = s.find('=');
+        if (first_non_space == std::string::npos || equals_index == std::string::npos) {
+            break;
```

Comment:
I'm fine with the break/continue, but I don't really fully grok what this is doing. I would sync up with @wgallagher